packages/@aws-cdk/aws-elasticsearch/README.md

```diff
@@ -21,18 +21,8 @@ To create an Elasticsearch domain:
 import * as es from '@aws-cdk/aws-elasticsearch';
 
 const domain = new es.Domain(this, 'Domain', {
-    elasticsearchVersion: es.Version.ES_7_1,
-    clusterConfig: {
```

Comment:
I think it would be good to have an example with the cluster configuration as well. What do you think?

Reply:
Agreed, that was one of my todos!

Reply:
I have added an example to the readme.